src/components/SecondaryNav/SecondaryNavHeader/SecondaryNavHeader.scss

```diff
@@ -0,0 +1,10 @@
+.secondaryNavHeader {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    min-height: 3em;
+
+    padding: 5px;
```

Comment:
We use a base 8 for spacing. So you can use `4px` or `8px`.
It's better if you use our spacing variables `var(--spacing-nano)` or `var(--spacing-small)`